gcp-repo/metadata.p2.inf

```diff
@@ -41,3 +41,27 @@ units.1.requires.1.greedy=true
 units.1.provides.1.namespace=org.eclipse.equinox.p2.iu
 units.1.provides.1.name=com.google.cloud.tools.eclipse.category.definition
 units.1.provides.1.version=1.0.0
+
+# Define the YEdit' category IU
+units.2.id=com.google.cloud.tools.eclipse.category
+units.2.version=$version$
+units.2.properties.1.name=org.eclipse.equinox.p2.type.category
+units.2.properties.1.value=true
+units.2.properties.2.name=org.eclipse.equinox.p2.name
+units.2.properties.2.value=YEdit
+units.2.properties.3.name=org.eclipse.equinox.p2.provider
+units.2.properties.3.value=Google Inc.
```

Comment:
Shouldn't be Google.

Reply:
I'm not sure. It's coming out of our repo now. It depends on exactly what "provider"means here. Per docs, "A property key (value "org.eclipse.equinox.p2.provider") for a String property containing information about the vendor or provider of the installable unit."

Reply:
I think Orbit would count as a provider, since they re-create the .jar as a bundle.  Since we're just redistributing, my 2¢ is that it should be _YEdit Project_ (as per the feature).

But we can sign it: a jar can have multiple signers.